library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -756,12 +758,14 @@ void vtkF3DRenderer::Render()
   int fps = static_cast<int>(std::round(
     1.0 / (std::chrono::duration_cast<std::chrono::microseconds>(cpuElapsed).count() * 1e-6)));
 
+#ifndef __ANDROID__
   glEndQuery(GL_TIME_ELAPSED);
   GLint elapsed;
   glGetQueryObjectiv(this->Timer, GL_QUERY_RESULT, &elapsed);
 
   // Get min between CPU frame per seconds and GPU frame per seconds
   fps = std::min(fps, static_cast<int>(std::round(1.0 / (elapsed * 1e-9))));
```

Comment:
is the computed fps on android correct without this ?

Reply:
Not entirely. The GPU fps is not computed, only the CPU one.

Reply:
good enough I guess